packages/react-storage/src/components/StorageImage/StorageImage.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+
+import { Image, ComponentClassNames } from '@aws-amplify/ui-react';
+import { useStorageURL } from '@aws-amplify/ui-react/internal';
+
+import type { StorageImageProps } from './types';
+
+export const StorageImage = ({
+  imgKey,
+  accessLevel,
+  identityId,
+  fallbackSrc,
```

Comment:
What is then expected behavior  if `imgKey`, `fallbackSrc` and `src` are all provided?